idr/externalDBs.py

```diff
@@ -90,3 +81,44 @@ def get_ensembleid(gene):
     for i in results.ix[:, 0]:
         ensembleid.append(i)
     return ensembleid
+
+
+def genes_of_interest_go(go_term, taxonomy_id):
+
+    url = ('http://www.ebi.ac.uk/QuickGO-Old/GAnnotation?tax=' +
+           taxonomy_id +
+           '&relType=IP&goid=%20' +
+           go_term +
+           '%20&format=tsv')
+    res = requests.get(url)
+    df = read_csv(StringIO(res.text), sep='\t', header=None)
+    c1 = df.iloc[:, 3]
+    genes = list(set(np.unique(c1.values.ravel())) - set(['Symbol', '-']))
+    if genes == []:
+        url = ('http://www.ebi.ac.uk/QuickGO-Old/GAnnotation?tax=' +
+               taxonomy_id +
+               '&goid=%20' +
+               go_term +
+               '%20&format=tsv')
+        res = requests.get(url)
+        df = read_csv(StringIO(res.text), sep='\t', header=None)
+        c1 = df.ix[:, 3]
+        genes = list(set(np.unique(c1.values.ravel())) - set(['Symbol', '-']))
+    return genes
+
+
+def ensembleid_to_genesymbol(ensembleid):
+
+    ensembleserver = "http://rest.ensembl.org/xrefs/id/"
+    url = (ensembleserver + ensembleid +
+           "?content-type=application/json;external_db=WikiGene")
```

Comment:
This works, but `requests.get` supports query parameters which are clearer, e.g.
```
url = 'http://rest.ensembl.org/xrefs/id/ENSG00000106526'
params = {'content-type': 'application/json', 'external_db': 'WikiGene'}
requests.get(url, params)
```